compute_tools/src/spec.rs

```diff
@@ -511,17 +524,19 @@ pub fn handle_grants(spec: &ComputeSpec, connstr: &str, client: &mut Client) ->
         .map(|r| r.name.pg_quote())
         .collect::<Vec<_>>();
 
-    for db in &spec.cluster.databases {
-        let dbname = &db.name;
+    if spec_has_webaccess(spec) {
+        for db in &spec.cluster.databases {
```

Comment:
Some comment would be useful. IIUC, all new roles will have `neon_superuser`, so it's not needed for them, wright?

Reply:
Not 100%, how backward compatibility will work in this case

1. If there is an existing project without web_access, its roles don't have `neon_superuser` granted
2. If we create a new db, existing roles won't have create on it, so they won't be able to create extensions

Reply:
Maybe we can grant `neon_superuser` to all roles at the moment we create `neon_superuser`?

Reply:
OK I changed it so that at the moment that we create `neon_superuser`, we also grant all privileges on all databases to `neon_superuser`, and we grant `neon_superuser` to all roles.